javascript-local-storage/data.js

```diff
@@ -0,0 +1,12 @@
+/* exported todos */
+
+var todos = [];
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+if (previousTodosJSON !== null) {
+  todos = [];
+}
+window.addEventListener('beforeunload', function (event) {
+  event.preventDefault();
```

Comment:
you do not need to use this here. Primarily it is used for forms to prevent refresh of the application.